src/inim.nim

```diff
@@ -23,10 +25,23 @@ const
     ]
     # preloaded code into user's session
     EmbeddedCode = staticRead("inimpkg/embedded.nim")
+    ConfigDir = getConfigDir() / "inim"
+    RcFilePath = ConfigDir / "inimrc"
```

Comment:
Should we call it `inim.ini`, as _"The configuration file's syntax is similar to the Windows .ini format"_, and then we'll get syntax highlighting for free?